src/ui/Pricing/SubscriptionSettings/SubscriptionCard/Card.svelte

```diff
@@ -147,15 +145,62 @@
     display: inline-flex;
     --bg: var(--primary, var(--green));
     --bg-hover: var(--primary-hover, var(--green-hover));
+    --v-padding: 8px;
   }
 
   .btn-2 {
     --bg: var(--white);
+    --v-padding: 7px;
+  }
+
+  .subaction {
+    margin-left: 12px;
   }
 
   .billing {
     position: absolute;
     bottom: 16px;
     right: 16px;
   }
+
+  .label {
+    font-size: 12px;
+  }
+
+  :global(.phone-xs),
+  :global(.phone) {
+    .actions {
+      --margin: 16px;
+      flex-direction: column;
+      align-items: flex-start;
+
+      button,
+      a {
+        font-size: 16px;
+        line-height: 24px;
+      }
+    }
+
+    .subaction {
+      margin-left: 0;
+      margin-top: 8px;
+    }
+
+    .label {
+      transform: translateY(25%);
+      font-size: 16px;
```

Comment:
Correct typography requires font-size and line-height to be applied accordingly.